Doc/whatsnew/3.13.rst

```diff
@@ -96,6 +103,7 @@ New typing features:
 
 * :pep:`696`: Type parameters (:data:`typing.TypeVar`, :data:`typing.ParamSpec`,
   and :data:`typing.TypeVarTuple`) now support defaults.
+
```

Comment:
This is probably on me but we need to mention PEP 705 here and PEP 696 (which is a syntax change) could use a more prominent mention.

Reply:
I've added a mention of PEP 705. I'm not sure what you want the entry for PEP 696 to say beyond what it says.